asm/non_matchings/code/z_skin_matrix/SkinMatrix_Invert.s

```diff
@@ -17,7 +17,7 @@ glabel D_80140844
     .float 0.0005
 
 .text
-glabel func_800A73E0
+glabel Skin_Matrix_Invert
```

Comment:
Shouldn't this be SkinMatrix_Invert? Nonmatching builds will currently complain as the nonmatching version of this function is already SkinMatrix_Invert but all references are Skin_Matrix_Invert.
Rest looks good otherwise.

Reply:
Good catch!